keyboards/clickety_split/leeloo/keymaps/default/keymap.c

```diff
@@ -92,23 +91,39 @@ const uint16_t PROGMEM keymaps[][MATRIX_ROWS][MATRIX_COLS] = {
  * ,-----------------------------------------.                                ,-----------------------------------------.
  * |      |      |      |      |      |      |                                |      |      |      |      |      |      |
  * |------+------+------+------+------+------|                                |------+------+------+------+------+------|
- * |      |      |      |      |      |      |                                |      |      |      |      |      |      |
+ * |      |      |      |      |      |      |                                |      |      | +VOL |      |      |      |
  * |------+------+------+------+------+------|                                |------+------+------+------+------+------|
- * |      |      |      |      |      |      |--------.               ,-------|      |      |      |      |      |      |
+ * |      |      |      |      |      |      |--------.               ,-------|      |      | -VOL |      |      |      |
  * |------+------+------+------+------+------|        |               |       |------+------+------+------+------+------|
- * |      |      |      |      |      |      |--------|               |-------|      |      |      |      |      |      |
+ * |      |      |      |      |      |      |--------|               |-------|      | MUTE |      |      |      |      |
  * `-----------------------------------------/       /                 \      \-----------------------------------------'
  *                | LALT  | LCTL  | ENT   | / MINS  /                   \ EQL  \  | SPC   | BSPC  |  DEL  |
  *                |       |       | LOWR  |/ RAISE /                     \RAISE \ | LOWR  |       |       |
  *                `-------------------------------'                       '------''-----------------------'
  */
   [_ADJUST] = LAYOUT(
+#if defined(KEYBOARD_clickety_split_leeloo_rev1)
```

Comment:
This should be removed.  

If RGB isn't enabled, then the keys don't do anything, and can be left here.  

Otherwise, there should probably be separate keymap folders.

Reply:
Thank you, @drashna,

I'll remove the condition and leave the RGB keycodes, yet I'll place a short comment in for Leeloo v1 boards.  Leeloo v2 keyboards have the RGB feature turned on because they have them installed.